variables.tf

```diff
@@ -3,6 +3,11 @@ variable "naming_rules" {
   type        = string
 }
 
+variable "dns_servers" {
+  description = "Custom DNS servers (if applicable) ,  list of custom DNS servers to use inside your virtual network instead of the Azure-provided resolver" 

```

Comment:
Sorry to nitpick but can you edit for readability and spacing here?

Suggestion: "If applicable, a list of custom DNS servers to use inside your virtual network instead of the Azure-provided resolver"

Reply:
no issues , done the changes , thank you for the checks